aiodocker/images.py

```diff
@@ -0,0 +1,220 @@
+import json
+import base64
+from typing import Optional, List, Dict, BinaryIO
+from .utils import clean_config
+from .jsonstream import json_stream_result
+
+
+class DockerImages(object):
+    def __init__(self, docker):
+        self.docker = docker
+
+    async def list(self, **params):
+        """
+        List of images
+        """
+        response = await self.docker._query_json(
+            "images/json", "GET",
+            params=params,
+            headers={"content-type": "application/json", },
+        )
+        return response
+
+    async def get(self, name):
+        """
+        Return low-level information about an image
+        Args:
+            name: name of the image
+        """
+        response = await self.docker._query_json(
+            "images/{name}/json".format(name=name),
+            headers={"content-type": "application/json", },
+        )
+        return response
+
+    async def history(self, name):
+        response = await self.docker._query_json(
+            "images/{name}/history".format(name=name),
+            headers={"content-type": "application/json", },
+        )
+        return response
+
+    async def pull(self,
+                   from_image: str,
+                   repo: Optional[str]=None,
```

Comment:
Yeah, I agree with that and can bear with this breaking changes in my projects.
It's good to have consistent styles/conventions with other aio-libs projects, particularly, aiohttp.